home/templates/home/djangonaut_survey_response.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}{{ djangonaut.get_full_name }}'s Application | Djangonaut Space{% endblock %}
+{% block meta_title %}{{ djangonaut.get_full_name }}'s Application | Djangonaut Space{% endblock %}
+
+{% block content %}
+<main class="section my-3 mx-5">
+  <div class="section-container max-w-4xl mx-auto">
+    <nav class="mb-4 text-sm" aria-label="breadcrumb">
+      <ol class="flex items-center gap-2">
+        <li><a href="{% url 'user_sessions' %}" class="text-blue-600 hover:text-blue-800 hover:underline">{% trans "My Sessions" %}</a></li>
+        <li class="text-gray-400">/</li>
+        <li><a href="{% url 'team_detail' session_slug=session.slug pk=team.pk %}" class="text-blue-600 hover:text-blue-800 hover:underline">{{ session.title }} - {{ team.name }}</a></li>
+        <li class="text-gray-400">/</li>
+        <li class="text-gray-600">{{ djangonaut.get_full_name|default:djangonaut.username }}'s Application</li>
+      </ol>
+    </nav>
+
+    <h1 class="text-4xl font-extrabold mb-2">{{ djangonaut.get_full_name|default:djangonaut.username }}</h1>
+    <p class="text-gray-600 mb-6">Application for {{ session.title }}</p>
+
+    <div class="bg-white shadow rounded-lg p-6">
+      <h2 class="text-2xl font-extrabold mb-4">Survey Responses</h2>
+
+      {% if question_responses %}
+        <dl class="space-y-4">
+          {% for question_response in question_responses %}
+            <div class="border-l-4 border-purple-500 pl-4 py-2">
+              <dt class="text-sm font-bold text-gray-700 mb-2">
+                {{ question_response.question.label }}
+                {% if question_response.question.help_text %}
+                  <span class="block text-xs font-normal text-gray-500 mt-1">{{ question_response.question.help_text }}</span>
```

Comment:
```suggestion
                  <span class="block text-xs font-normal text-gray-500 mt-1">{{ question_response.question.help_text|linebreaksbr|urlize }}</span>
```

This really should reuse the detail.html template